src/content/docs/fr/guides/integrations-guide/mdx.mdx

```diff
@@ -1,6 +1,8 @@
 ---
```

Comment:
We miss some changes in this file:
* the [`MDX in Astro`](https://github.com/withastro/docs/blob/5d7321df418b5dfa0d0c51e04f84535914bd8626/src/content/docs/en/guides/integrations-guide/mdx.mdx?plain=1#L91) section
* removing a useless `await` (see #10332)

Reply:
The `MDX In Astro` [section is already written](https://github.com/thomasbnt/docs/blob/ebe4f7be40f353a58f2cea6dc75a39c6ba1c6c51/src/content/docs/fr/guides/integrations-guide/mdx.mdx?plain=1#L91), what I miss?

Reply:
Oh sorry, I linked the wrong section... It was [`Using MDX wth content collections`](https://github.com/withastro/docs/blob/5d7321df418b5dfa0d0c51e04f84535914bd8626/src/content/docs/en/guides/integrations-guide/mdx.mdx?plain=1#L99) just below 😅

Reply:
Just found the missing PR: #10187 😉

Reply:
Just pushed here, thanks @ArmandPhilippot. Also have a little error link from the file `fr/guides/content-collections/#defining-the-collection-loader`.